### ISSUE TITLE
Cannot build from SVN.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [openroc/aforge#251]
```
What steps will reproduce the problem?
1.  Download trunk from svn.
2.  Attempt to build with VS2008.
3.  You should get 500+ compile errors.

What is the expected output? What do you see instead?
The project should build from SVN without any additional work on my behalf.

What version of the product are you using?
Trunk.

Please provide any additional information below.
The reason for this is dependencies in the project are configured incorrectly.  
For example, the "Genetic" library needs "AForge" and AForge.Math", but under 
References in VS2008 there's a warning.  Removing the bunk reference and adding 
a reference to Core and Math resolves the issue.  

Basically, the way the project is setup prevents people from cleanly building 
sources.  is there some reason the project it doing this, rather than just 
using the build-in project reference handling?
```

Original issue reported on code.google.com by `kid...@gmail.com` on 6 Sep 2011 at 1:41
